tools/etcd-dump-metrics/etcd.go

```diff
@@ -53,11 +53,15 @@ func setupEmbedCfg(cfg *embed.Config, curls, purls, ics []url.URL) {
 	cfg.ListenClientUrls, cfg.AdvertiseClientUrls = curls, curls
 	cfg.ListenPeerUrls, cfg.AdvertisePeerUrls = purls, purls
 
-	cfg.InitialCluster = ""
-	for i := range ics {
-		cfg.InitialCluster += fmt.Sprintf(",%d=%s", i, ics[i].String())
```

Comment:
It seems an unnecessary change,
- The function `setupEmbedCfg` is already very simple. 
- The new function `joinURLs` isn't used by multiple users (it has only one user).